quic/s2n-quic-core/src/datagram/unreliable_datagram.rs

```diff
@@ -1,6 +1,8 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
+use crate::time::Timestamp;
```

Comment:
I think we should probably use `crate::event::Timestamp`, since it has better stability guarantees in the API. The `crate::time::Timestamp` should really be only used internally.